iGCS/Utils/SettingsData.h

```diff
@@ -0,0 +1,31 @@
+//
+//  SettingsData.h
+//  iGCS
+//
+//  Created by David Leskowicz on 2/14/15.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+typedef enum {
```

Comment:
Please use NS_ENUM macro to define. see http://nshipster.com/ns_enum-ns_options/